hetzner_ddns.sh

```diff
@@ -13,7 +13,24 @@ for arg in $(seq "$#"); do
             printf '%s %s\n' "$self" "$version"
             exit 0;;
         '--help'|'-h')
-            man hetzner_ddns;
+            # man usually not available on OpenWrt (too big)
+            # use short help string in that case
+            if ! command -v man > /dev/null
+            then
+                printf '%s\n\n%s\n\n' "hetzner_dns 0.2.5 (2 August 2024) - Hetzner Dynamic DNS Daemon" "Usage hetzner_dns.sh [--daemon] [CONFIG]"
```

Comment:
I think that using `printf` with string formatting string is superfluous here. While it loses indentation, the snipper below is somewhat shorter and faster:
```sh
if ! command -v man > /dev/null; then
    echo "${self} ${version} - Hetzner Dynamic DNS Daemon
Usage hetzner_dns.sh [--daemon] [CONFIG]
This program runs as system service /etc/init.d/hetzner_ddns on OpenWrt.
                -h, --help     Print help and exit
                -v, --version  Print version and exit
                -d, --daemon Detach from current shell and run as a deamon
Configuration file is located at /usr/local/etc/hetzner_ddns.conf
    interval='60'                           # Seconds between updates / TTL value
    key='********************************'  # Hetzner DNS API key
    domain='example.com'                    # Top level domain name
    records='homelab media vpn'             # Space separated host subdomains (@ for domain itself)
Control service with: service hetzner_ddns status|start|stop|restart after specifying the config"
else
    man hetzner_ddns;
fi
```

Reply:
:grimacing: I agree, thanks for that suggestion. I tried to fix that with [a218e96](https://github.com/filiparag/hetzner_ddns/pull/18/commits/a218e968d6d4326a58d6092d5e7db56aa38a07d4)